### DISCUSSION TITLE
Add new metrics?

### DISCUSSION THREAD

**Discussion by LrrrAc** [anclrii/Storj-Exporter#67] [Q&A] [answered]
Is there a way to add new metrics to be collected? Im specifically looking to have "storj_payout_currentMonthExpectations" collected.

**Comment by anclrii** [comment, marked as answer]
Some metrics are not available in node API, I'll check if this one is. If it's there I'll add it.

**Reply by LrrrAc** [reply]
Thanks so much! I find it at "https://DOMAIN.com/api/sno/estimated-payout".

**Comment by anclrii** [comment]
Hey @LrrrAc I added it in https://github.com/anclrii/Storj-Exporter/releases/tag/2.1.2. 
If you use docker you can just pull `latest`.

It's exported like so:

```
storj_payout_currentMonth{type="currentMonthExpectations"} 963.0
```

I think I'll add it to my grafana dashboard as well.

**Reply by LrrrAc** [reply]
Thank you! It works perfectly!

**Comment by anclrii** [comment]
Added also in grafan dashboard https://github.com/anclrii/Storj-Exporter-dashboard or https://grafana.com/grafana/dashboards/13810
